app/build.gradle.kts

```diff
@@ -8,23 +8,19 @@ plugins {
 
 android {
     namespace = "com.boolder.boolder"
-    compileSdk = 32
+    compileSdk = 33
```

Comment:
what does this do?

Reply:
This is the version of the android SDK that is used to compile the application. The update was needed in order to use the last version of Jetpack compose.

Otherwise, some errors like the following are happening when compiling:
```
Dependency 'androidx.compose.material3:material3:1.0.1' requires libraries and applications that
depend on it to compile against version 33 or later of the
Android APIs.

:app is currently compiled against android-32.

Recommended action: Update this project to use a newer compileSdk
of at least 33, for example 33.

Note that updating a library or application's compileSdk (which
allows newer APIs to be used) can be done separately from updating
targetSdk (which opts the app in to new runtime behavior) and
minSdk (which determines which devices the app can be installed
on).
```
☝️ Some precisions are included about the `targetSdk` and the `compileSdk` 😉